ister.py

```diff
@@ -103,7 +103,7 @@ def run_command(cmd, raise_exception=True, log_output=True, environ=None,
                 LOG.debug(decoded_line)
 
         if proc.poll() and raise_exception:
-            if len(stderr) > 0:
```

Comment:
style fix-up/changes should go in a separate commit from the one that changes or adds functionality.